packages/ui/src/views/repo/pull-request/details/components/conversation/sections/pull-request-merge-section.tsx

```diff
@@ -49,145 +88,115 @@ const PullRequestMergeSection = ({
-                    : !mergeable
-                      ? 'Conflicts found in this branch'
-                      : `This branch has no conflicts with ${pullReqMetadata?.target_branch} branch`
-                }
-                icon={
-                  unchecked ? (
-                    // TODO: update icon for unchecked status
-                    <Icon name="clock" className="text-warning" />
-                  ) : (
-                    <>
-                      {mergeable ? (
-                        <Icon name="success" className="text-foreground-success" />
-                      ) : (
-                        <Icon name="triangle-warning" className="text-destructive" />
-                      )}
-                    </>
-                  )
-                }
-              />
-            }
-            description={
-              unchecked ? (
-                <LineDescription text={'Checking for ability to merge automatically...'} />
-              ) : !mergeable ? (
-                <div className="ml-6 inline-flex items-center gap-2">
-                  <Layout.Vertical>
-                    <p className="text-14 font-normal text-foreground-4">
-                      Use the
-                      {/* TODO: looks like below should be a Link component (<StyledLink variant="accent" />) */}
+    <Accordion.Item value={ACCORDION_VALUE} isLast>
+      <Accordion.Trigger
+        className="py-3 text-left [&>svg]:-rotate-0 [&>svg]:data-[state=open]:-rotate-180"
+        chevronClassName="text-icons-3 self-start mt-1"
+        hideChevron={mergeable || unchecked}
+      >
+        <StackedList.Field
+          className="flex gap-y-1"
+          title={
+            <LineTitle
+              textClassName={isConflicted ? 'text-foreground-danger' : ''}
+              text={
+                unchecked
+                  ? 'Merge check in progress...'
+                  : !mergeable
+                    ? 'Conflicts found in this branch'
+                    : `This branch has no conflicts with ${pullReqMetadata?.target_branch} branch`
+              }
+              icon={
+                unchecked ? (
+                  <Icon name="clock" className="text-icons-alert" />
+                ) : (
+                  <Icon
+                    className={mergeable ? 'text-icons-success' : 'text-icons-danger'}
+                    name={mergeable ? 'success' : 'triangle-warning'}
+                  />
+                )
+              }
+            />
+          }
+          description={
+            <>
+              {unchecked && <LineDescription text={'Checking for ability to merge automatically...'} />}
+              {isConflicted && (
+                <Layout.Vertical className="ml-6">
+                  <p className="text-14 font-normal text-foreground-4">
+                    Use the&nbsp;
+                    <Button variant="link_accent" size="text" onClick={handleCommandLineClick} asChild>
```

Comment:
Any reason we couldn't just kill off the child and drop the `asChild` entirely? Instead of trying to get a span to act like a button, can we just use the button?

<img width="843" alt="Screenshot 2025-03-25 at 15 22 03" src="https://github.com/user-attachments/assets/199f016f-5866-47e6-a1fd-cb81ce3f9100" />